source/content/drupal-9.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Drupal 9 Early Access
+description: Test the latest Drupal on the Pantheon Website Management Platform.
+categories: [get-started]
+tags: [site]
+earlyaccess: true
+contributors: [populist, edwardangert]
+reviewed: "2020-05-21"
+---
+
+Drupal 9 is, according to [drupal.org](https://www.drupal.org/docs/understanding-drupal/drupal-9-resources), "a cleaned-up version of Drupal 8... \[with] deprecated code removed and third-party dependencies updated."
+
+Drupal 9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://symfony.com/releases/4.4) and [Twig 2](https://twig.symfony.com/doc/2.x/index.html), removes several deprecated API functions in favor of better options, and allows everyone running Drupal 8.8+ an easy upgrade path to Drupal 9 & beyond.
+
+<Alert title="Warning" type="danger">
+
+**Do not use this, or any, Early Access release for your live site.**
+
+Currently, Drupal 9 is only available on Pantheon as an Early Access feature release. This means the feature is currently in active development and is intended for testing and evaluation purposes only.
+
+Pantheon engineers are rolling out changes often, and easy updates from Drupal 9 Early Access are not guaranteed.
+
+**Sites may be deleted without notice**.
+
+</Alert>
+
+Choose one of the following methods to get started with a new Drupal 9 site to [create a new Drupal 9 site with integrated Composer](#create-a-new-drupal-9-site-with-integrated-composer) or to test your existing site's [compatibility with an upgrade](#test-an-existing-drupal-site-for-drupal-9-upgrade-compatibility).
```

Comment:
It might be helpful to give people some guidance or advice on which option to select.

- Using the "Drupal 9 Early Access" form is best for those who want to try out the new Integrated Composer feature. This is also probably the best option right now for folks who just want to try Drupal 9 with the least overhead.
- Use `build:project:create` if already using the Build Tools workflow. I'd also say this option was best for those interested in automated testing, but I don't think any automated tests are working yet.
- Even though we don't have a section for the drops-8 branch yet, we might advise those not using Composer just wait & use the branch once it becomes available.

This advice could either come before the options, after the options, or be divided up so that each section has the blerb about when it is most applicable to use it.

Reply:
+1, I recommend rewrite this sentence as something like 

> You can test Drupal 9 now with a fresh installation using either our Integrated Composer Early Access program or Build Tools. We recommend Integrate Composer if you simply want to test the browser experience of Drupal 9 and get access to forthcoming Pantheon support for Composer in the Dashboard. Use Build Tools if you want to create a continuous integration workflow using GitHub. If you are more concerned with upgrading an existing site, you can check compatibility with an upgrade.

Reply:
How about copying the table from the blog post into here?

Reply:
pushed a commit with this in bullet points - testing with the table while it builds a preview

Reply:
table added as well